airflow/configuration.py

```diff
@@ -459,7 +459,7 @@ def run_command(command):
 plugins_folder = {TEST_PLUGINS_FOLDER}
 base_log_folder = {AIRFLOW_HOME}/logs
 executor = SequentialExecutor
-sql_alchemy_conn = sqlite:///{AIRFLOW_HOME}/unittests.db
+sql_alchemy_conn_cmd = sh -c 'echo "sqlite:///{AIRFLOW_HOME}/unittests.db"'
```

Comment:
> @thesquelched you are changing the default configuration to a "_cmd". That seems not required for this PR. Am I overlooking something? Otherwise please remove that change.

I've changed `TEST_CONFIG`, which as far as I can tell, is only loaded during unit tests.  And the two lines are functionally equivalent, except that my change exercises the `_cmd` functionality, which isn't tested anywhere else currently.